src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/SphKerrSchild.hpp

```diff
@@ -0,0 +1,468 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <pup.h>
+
+#include "DataStructures/CachedTempBuffer.hpp"
+#include "DataStructures/Tags/TempTensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/AnalyticSolution.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Solutions.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/ForceInline.hpp"
+#include "Utilities/MakeArray.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace Tags {
+template <typename Tag>
+struct dt;
+}  // namespace Tags
+namespace gsl {
+template <class T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace gr {
+namespace Solutions {
+
+class SphKerrSchild : public AnalyticSolution<3_st>,
```

Comment:
add documentation?

Rename to `SphericalKerrSchild`.  This function will be helpful: https://github.com/nilsdeppe/MyEnvironment/blob/399b074119e183709b15f11a66a2729971cdfda1/.zshrc#L270